fabric-resource-loader-v0/src/main/java/net/fabricmc/fabric/api/resource/ResourceManagerHelper.java

```diff
@@ -52,4 +54,47 @@ default void addReloadListener(IdentifiableResourceReloadListener listener) {
 	static ResourceManagerHelper get(ResourceType type) {
 		return ResourceManagerHelperImpl.get(type);
 	}
+
+	/**
+	 * Registers a built-in resource pack.
```

Comment:
This should document that packs added with this overload aren't enabled by default.

Reply:
Would `Registers a built-in resource pack. That resource pack is not enabled by default.` enough/good?